### PR TITLE
Make chapter list in README clickable

### PR DESCRIPTION
**Pull request by fmueller**
To make it more convenient for readers to jump directly to specific chapters from README on Github. Also added travis-ci badge to see if build is working properly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,22 +14,24 @@ Feel free to use these guidelines as a guidance for your own development. Note t
...
-- Performance
-- Pagination
-- Hypermedia
-- Data Formats
-- Common Data Objects
-- Common Headers
-- Proprietary Headers
-- API Discovery
-- Tooling
+- [Design Principles](http://zalando.github.io/restful-api-guidelines/design-principles/DesignPrinciples.html)
```
Comment (whiskeysierra) [inline README.md]: Would it make sense to link to the source file instead of the page here?

Comment (fmueller) [inline README.md]: The intention is to link to the generated Gitbook.

**Comment by whiskeysierra** [comment]
:+1:

**Comment by ePaul** [comment]
:+1:

**Pull request merged**